src/styles/components/_new-release-banner.scss

```
@@ -0,0 +1,74 @@
+@import '../abstracts/_abstracts-dir';
+
+.new-release-banner {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: rem(130px);
+  padding: rem(16px 16px);
+
+  background: {
+    image: url('/images/new-release.png');
+    position: center;
+    repeat: no-repeat;
+    size: cover;
+  }
+
+  @include respond-above('md') {
+    height: rem(220px);
+    padding: rem(40px 40px);
+  }
+
+  @include respond-above('lg') {
+    height: rem(300px);
+    padding: rem(44px 44px);
+  }
+
+  @include respond-above('xl') {
+    height: rem(370px);
+    padding: rem(52px 52px);
```

Same [here](https://github.com/Cobda/cobda-web/pull/37/files#r586362425)
```suggestion
  @include respond-above('md') {
    height: rem(220px);
    padding: rem(40px);
  }

  @include respond-above('lg') {
    height: rem(300px);
    padding: rem(44px);
  }

  @include respond-above('xl') {
    height: rem(370px);
    padding: rem(52px);
```